corehq/apps/mocha/README.md

```diff
@@ -1,6 +1,33 @@
 Mocha Tests
 ===========
 
+## Getting started
+
+1. Install `grunt-mocha`
+
+```
+npm install grunt-mocha
```

Comment:
this should get installed when one runs `npm install` (along with all other deps, so i recommend using that)